internal/gonut/cmd/push.go

```diff
@@ -208,18 +210,39 @@ func runSampleAppPush(app sampleApp) error {
 		return fmt.Errorf("unsupported delete setting: %s", deleteSetting)
 	}
 
+	// If summary is set to full, enable pinging to display it in the summary.
```

Comment:
I do not like that we mix logic of the summary with checks. To be honest, I would like to remove this implicit enablement.